cvpy/tests/test_imagetable.py

```diff
@@ -34,8 +34,7 @@ class TestImageTable(unittest.TestCase):
     DATAPATH = None
 
     def setUp(self) -> None:
-        self.s = CAS(TestImageTable.CAS_HOST, TestImageTable.CAS_PORT, TestImageTable.USERNAME,
-                     TestImageTable.PASSWORD)
+        self.s = CAS(TestImageTable.CAS_HOST, TestImageTable.CAS_PORT, protocol='http')
```

Comment:
Hi Pankaj, is there a reason you removed the username and password here? I know Jordan and I were having trouble running these tests with protocol='http'. I wonder if we need different swat calls to differentiate from Windows/Mac.

Reply:
I made that change for my local testing. I reverted it back to include username and password, and removed http protocol. Thanks for pointing it out.